### PR TITLE
Updated Lifecycle.md to improve flow

### PR DESCRIPTION
**Pull request by victoronline**
This is a first pass. Updated to improve flow.

### CODE REVIEW

File: Lifecycle.md [hunk 1]
```diff
@@ -34,12 +34,12 @@ chartered by the TC of the project they exist under.
...
-	3. That TC submits an application to the Planetary Software TSC
-	4. If the application is accepted by the TSC, the TL* enters Incubation
-	5. The TSC works with the TC to get the TL* graduated from Incubation
-	6. The TL* is now a fully authorized Planetary Software TL*
+	* Like-minded individuals form a TC in order to establish a TL*
+	* That TC establishes a charter for their TL* and submits an application to the Planetary Software TSC
+	* If the application is accepted by the TSC, the TL* enters Incubation
+	* A mentor(s) is assigned tor work the TC to get the TL* to ensure requirements are met
+	* Once requirements are met, the TSC votes to graduate the TL* from Incubation
+	* The TL* is now a fully authorized Planetary Software TL*
```
Comment (rbeyer) [inline Lifecycle.md]: I know these were like this in the parent document (but I think that was an oversight), these probably don't need to be verbatim text, and probably can be 'regular'.

**Review by jessemapel — COMMENTED** [review]
I agree with Ross about the formatting. Content looks great :+1:

File: Lifecycle.md [hunk 2]
```diff
@@ -34,12 +34,12 @@ chartered by the TC of the project they exist under.
...
-	1. Like-minded individuals form a TC in order to establish a TL*
-	2. That TC establishes a charter for their TL*
-	3. That TC submits an application to the Planetary Software TSC
-	4. If the application is accepted by the TSC, the TL* enters Incubation
-	5. The TSC works with the TC to get the TL* graduated from Incubation
-	6. The TL* is now a fully authorized Planetary Software TL*
+	* Like-minded individuals form a TC in order to establish a TL*
+	* That TC establishes a charter for their TL* and submits an application to the Planetary Software TSC
+	* If the application is accepted by the TSC, the TL* enters Incubation
+	* A mentor(s) is assigned tor work the TC to get the TL* to ensure requirements are met
```
Comment (jlaura) [inline Lifecycle.md]: /s/tor/to/

File: Lifecycle.md [hunk 3]
```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
...
+limited resources available to mentor new projects. As such, projects
+are chosen for admission as mentors become available.
+
+You can apply at any time and the TSC and available mentors will
+help improve your application while awaiting available resources.
+
+The process to join the Planetary Software Organization
 as a TLP or TLWG is as follows.
 
+### 1. Application Completed - Pull Request Submitted
```
Comment (jlaura) [inline Lifecycle.md]: I am finding the 'Application Completed' to be weird here. If the PR is submitted is the application complete? It seems like that just starts the process.?

File: Lifecycle.md [hunk 4]
```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
...
+
 Copy the [Application Template](Applications/Template.md), fill
 it out, add the file to the [Applications](Applications/)
-directory, and submit that as a Pull Request.
+directory and submit this as a Pull Request.
```
Comment (jlaura) [inline Lifecycle.md]: I'm not a fan of this/that here. What about 'and submit the filled in Template.md as a Pull Request to this repository'?

File: Lifecycle.md [hunk 5]
```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
...
 
 Applications do not have to be complete to be submitted, the TSC
 can work with the authors and their respective communities in each
 Pull Request.
 
-The Application is considered 'accepted' when the Pull Request is
-merged by the TSC.
+### 2. Pull Request is Merged - Application is Accepted
```
Comment (jlaura) [inline Lifecycle.md]: Maybe the above 'Application Complete' could be 'Application Submission' or something? Then #2 makes more sense I think.

File: Lifecycle.md [hunk 6]
```diff
@@ -262,7 +260,7 @@ meeting).
 
 The TLP or TLWG is considered to be chartered once that PR lands.
 
-### Badges
+#### Badges
```
Comment (jlaura) [inline Lifecycle.md:266]: badges - the reason we are all in this thing.

File: Lifecycle.md (hunk 1 above)
Comment (victoronline) [reply Lifecycle.md]: OK. Do we care about numbering these steps or just having them in order? I originally removed the numbers to not confuse with the numbered steps I added below.

Comment (rbeyer) [reply Lifecycle.md]: I don't think so.  If you think they look better as bullets, go for that.

File: Lifecycle.md (hunk 3 above)
Comment (rbeyer) [reply Lifecycle.md]: Yeah, maybe "Submit Application" here?

File: Lifecycle.md (hunk 1 above)
Comment (victoronline) [reply Lifecycle.md]: Ok. Changed but kept bullets.

File: Lifecycle.md (hunk 3 above)
Comment (victoronline) [reply Lifecycle.md]: Updated. Do we like the wording?

File: Lifecycle.md (hunk 4 above)
Comment (victoronline) [reply Lifecycle.md]: Added suggested text

File: Lifecycle.md (hunk 5 above)
Comment (victoronline) [reply Lifecycle.md]: Changed text to 'Submit Application'

**Review by rbeyer — APPROVED** [review]

**Review by jlaura — APPROVED** [review]

**Pull request merged**